bundle/community/manifests/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -531,6 +531,10 @@ spec:
                 - --zap-log-level=info
                 - --zap-time-encoding=rfc3339nano
                 - --enable-nginx-instrumentation=true
+                - '--target-allocator-image=ghcr.io/open-telemetry/opentelemetry-operator/target-allocator:'
+                - '--operator-opamp-bridge-image=ghcr.io/open-telemetry/opentelemetry-operator/operator-opamp-bridge:'
+                - --target-allocator-image=ghcr.io/open-telemetry/opentelemetry-operator/target-allocator:vlatest
+                - --operator-opamp-bridge-image=ghcr.io/open-telemetry/opentelemetry-operator/operator-opamp-bridge:vlatest
```

Comment:
Can you run `make reset` and then `make update` to get rid of these changes?

Reply:
That did it, thanks!